src/libstore/globals.hh

```diff
@@ -206,7 +206,17 @@ public:
 
     Setting<std::string> builders{
         this, "@" + nixConfDir + "/machines", "builders",
-        "A semicolon-separated list of build machines, in the format of `nix.machines`."};
+        R"(
+          A semicolon-separated list of build machines, where each machine follows this format:
+
+          {protocol}://{user}@{host} [{comma sep. systems} - {maxJobs} {speedFactor} {comma sep. features}]
+
+          Examples:
+          
+          ssh://root@builder1.com
+          
+          ssh://root@builder2.com x86_64-linux,aarch64-linux - 40 20 nixos-test,benchmark,big-parallel,kvm
```

Comment:
This should be formatted as markdown. Also it might be better to refer to the chapter on remote building in the manual rather than duplicate that information here.

Reply:
Oh, I wasn't aware that there is a manual chapter about this. Referencing it sounds like a good Idea. Is there any way to ensure that the exact version of the manual is referenced which matches the current nix version?

Reply:
Something like `[the manual chapter on remote builds](../advanced-topics/distributed-builds.md)` should work in the mdbook manual.

Reply:
Thanks. I did that now. Afterwards I checked the resulting mdbook and the link works.